### PR TITLE
Book.java is changed

### PR DESCRIPTION
**Pull request by Sarbertultra**
Changes **are not important**
```java
System.out.println ("Vot");
```

### CODE REVIEW

File: src/Book.java [hunk 1]
```diff
@@ -29,9 +30,7 @@ public void up(int raiting) {
               }
 
         public void append(String moreText){
-                   // Scanner scanner = new Scanner(System.in);
-            //System.out.println("Vvedite text knigi " + name);
-           // moreText = scanner.nextLine();
+
```
Comment (Sarbertultra) [inline src/Book.java:33]: hjhbjbhjbuh

File: src/Book.java [hunk 2]
```diff
@@ -29,9 +30,7 @@ public void up(int raiting) {
               }
 
         public void append(String moreText){
-                   // Scanner scanner = new Scanner(System.in);
-            //System.out.println("Vvedite text knigi " + name);
-           // moreText = scanner.nextLine();
```
Comment (Sarbertultra) [inline src/Book.java:34]: 46546448

File: src/Book.java [hunk 3]
```diff
@@ -18,6 +18,7 @@ public Book(String name, String author, int year, int raiting, String text) {
 
     public void up(int raiting) {
         System.out.println ("Nazvanie knigi "+ name+", "+" Nazvanie avtora "+author+ ", "+ " God vipuska "+ year+", "+"reiting " +3+", "+ "Tekst knigi "+":"+text);
+        System.out.println ("Vot");
```
Comment (Sarbertultra) [inline src/Book.java:21]: hniluhiugiu

**Comment by onlydianaT** [comment]
hjghjghkk

**Pull request closed without merging**